dropbox-sdk-dotnet/Dropbox.Api/DropboxRequestHandler.cs

```diff
@@ -528,13 +528,13 @@ private string CheckForError(string text)
                         return new Result
                         {
                             IsError = false,
-                            ObjectResult = await response.Content.ReadAsStringAsync()
-                        };
+                            ObjectResult = await response.Content.ReadAsStringAsync().ConfigureAwait(false)
+						};
```

Comment:
Can you change tab to whitespace here?

Reply:
Sure, sorry did this on my work laptop and they use TABs 😱